.github/scripts/changelog_checker

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+set -ue
+
+UNTRACKED="
```

Comment:
It's slightly confusing terminology, given that "untracked" has special meaning for Git. Perhaps "INFRASTRUCTURE", or "INTERNAL"?